packages/sources/cfbenchmarks/src/adapter.ts

```diff
@@ -29,7 +29,7 @@ export const makeWSHandler = (config?: Config): MakeWSHandler => {
   }
   const getSubscription = (type: 'subscribe' | 'unsubscribe', id?: string) => {
     if (!id) return
-    return { type, id, stream: 'value' }
+    return { type, id, stream: 'value', cacheID: id } // Temporarily add another key named cacheID that won't be ignored when generating the subscription key in getSubsID
```

Comment:
I've created this ticket to handle this better in the future. https://app.shortcut.com/chainlinklabs/story/24721/gracefully-handle-cases-where-ws-subscription-payload-messages-contains-ignored-key.

Reply:
Have you checked if this would affect any other adapters?

Reply:
It looks like every other definition of `getSubscription` for EAs uses a single asset pair, so I'm guessing this is unique to `cfbenchmarks`.

Reply:
yup it's for `cfbenchmarks` only.  It doesn't affect other adapters.